src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -19,6 +28,12 @@ const CrossPodQueryForm = () => {
   const { state, dispatch } = useStatusNotification();
   const { clearValue: clearUsername, ...username } = useField('text');
   const { selectedUser, setSelectedUser } = useContext(SelectUserContext);
+  const [documentType, setDocumentType] = useState('');
+  const [searchUser, setSearchUser] = useState('');
```

Comment:
We already have state set up for username (see Line 29), we do not need another useState for filling out that form.